corehq/apps/reports/static/reports/js/case_details.js

```diff
@@ -20,13 +20,13 @@ hqDefine("reports/js/case_details", function() {
             return self.itemsPerColumn * self.columnsPerPage();
         });
         self.columnClass = ko.observable('');
-        self.modalClass = ko.observable('');
-        self.modalDialogClass = ko.observable('');
+        self.isLargeModal = ko.observable(false);
+        self.isMediumModal = ko.observable(false);
```

Comment:
any reason you went with `Large` and `Medium` instead of `FullScreen` and `Large`? `'modal-lg': isMediumModal()` broke my brain a little bit.

Reply:
ha, point taken. updated.

Reply:
thanks!